### ISSUE TITLE
Thanks and Inquiry about DepthLM and Qwen-VL Trained Weights

### ISSUE THREAD

**Issue by yangcaoai** [facebookresearch/DepthLM_Official#10]
Thanks the authors for the great project. Using VLMs for metric depth estimation is a highly imaginative and impactful direction, and your work has significantly influenced my research thinking. Thank you to you and your team for the valuable contributions through the paper and the open-source code.

May I ask if you can also release the model weights trained on Qwen-VL? I would like to use them in my research project and cite your work in my paper. If there are any usage restrictions or license terms, I will of course follow them strictly.

Thank you again for your time and contributions. I look forward to your response.

Best,
Yang

**Comment by ZhipengCai** [comment]
thx for the interest! unfortunately we cannot release qwen-based models due to policy reasons. I would recommend you try out the pixtral model which provides reasonable accuracy. Meanwhile, for qwen-based models, we have released the training code to fully reproduce our result. Hope these alternatives can help, you can let us know if you meet further issues. And good luck to your research!

**Comment by yangcaoai** [comment]
> models

Understand. Thanks a lot for your quick reply :)

**Comment by ZhipengCai** [comment]
> > models
> 
> Understand. Thanks a lot for your quick reply :)

Good news! We are working on the release of 3b/7b model, it should come in a couple weeks. Stay tuned!
